doc/source/ray-overview/learn-more.md

```diff
@@ -1,8 +1,25 @@
 # Learn More
 
-Here are some talks, papers, and press coverage involving Ray and its libraries.
+Here are some general rules of thumb, talks, papers, and press coverage involving Ray and its libraries.
 Please raise an issue if any of the below links are broken, or if you'd like to add your own talk!
 
+## Latency numbers every Ray programmer should know
+
+| Stat | Value |
+| --- | --- | 
+| Typical Ray cluster startup time | ~6 minutes |
+| Max size of Ray cluster | ~500 machines |
```

Comment:
I think we can remove the scalability related numbers from this table. Instead, we can link to the "Scalability Envelope" https://github.com/ray-project/ray/blob/549527687f4bbfdaf5854535f8e7642e8eb3b643/release/benchmarks/README.md


As a follow on work, maybe we should move the entire Scalability Envelope into docs.ray.io

Reply:
I agree - it feels like it would make sense to include the metrics in the [Scalability Envelope](https://github.com/ray-project/ray/blob/549527687f4bbfdaf5854535f8e7642e8eb3b643/release/benchmarks/README.md) somewhere more immediately visible (and searchable) for Ray users. cc: @stephanie-wang @ericl 

Related issue: [#22551](https://github.com/ray-project/ray/issues/22551)